src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,13 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+
 public class UserRequestDto {
+    @NotNull
```

Comment:
u check it for null value in the `isValid()` method, so lets remove this annotation